libs/checkpoint-postgres/langgraph/store/postgres/base.py

```diff
@@ -1320,12 +1320,12 @@ def _ensure_index_config(
     index_config = index_config.copy()
     tokenized: list[tuple[str, Union[Literal["$"], list[str]]]] = []
     tot = 0
-    text_fields = index_config.get("text_fields") or ["$"]
-    if isinstance(text_fields, str):
-        text_fields = [text_fields]
-    if not isinstance(text_fields, list):
-        raise ValueError(f"Text fields must be a list or a string. Got {text_fields}")
-    for p in text_fields:
+    fields = index_config.get("fields") or ["$"]
```

Comment:
@hinthornw could you take a look the PostgresIndexConfig seems to indicate that "text_fields" is not a thing. Is this for backwards compatibility or just a typo?